nncf/quantization/algorithms/accuracy_control/ranker.py

```diff
@@ -80,33 +80,23 @@ class Ranker:
 
     def __init__(
         self,
-        ranking_subset_size: int,
```

Comment:
It looks like, this parameter was relevant for the `Ranker`.

Reply:
Fixed